spec.html

```diff
@@ -25098,7 +25130,7 @@ <h1>parseInt ( _string_, _radix_ )</h1>
             1. Set _R_ to 16.
         1. If _S_ contains a code unit that is not a radix-_R_ digit, let _Z_ be the substring of _S_ consisting of all code units before the first such code unit; otherwise, let _Z_ be _S_.
         1. If _Z_ is empty, return *NaN*.
-        1. Let _mathInt_ be the mathematical integer value that is represented by _Z_ in radix-_R_ notation, using the letters <b>A</b>-<b>Z</b> and <b>a</b>-<b>z</b> for digits with values 10 through 35. (However, if _R_ is 10 and _Z_ contains more than 20 significant digits, every significant digit after the 20th may be replaced by a 0 digit, at the option of the implementation; and if _R_ is not 2, 4, 8, 10, 16, or 32, then _mathInt_ may be an implementation-dependent approximation to the mathematical integer value that is represented by _Z_ in radix-_R_ notation.)
+        1. Let _mathInt_ be the mathematical integer value that is represented by _Z_ in radix-_R_ notation, using the letters <b>A</b>-<b>Z</b> and <b>a</b>-<b>z</b> for digits with values 10 through 35. (However, if _R_ is 10 and _Z_ contains more than 20 significant digits, every significant digit after the 20th may be replaced by a 0 digit, at the option of the implementation; and if _R_ is not 2, 4, 8, 10, 16, or 32, then _mathInt_ may be implementation-approximated to the mathematical integer value that is represented by _Z_ in radix-_R_ notation.)
```

Comment:
Also reads a bit strangely to me. Perhaps

> `_mathInt_` may be an implementation-approximated representation of the mathematical integer value

Or you could say "an implementation-approximation of", which isn't really any less defined than this use of "implementation-approximated" as the past participle of a verb (since its definition, despite being spelled the same, is for an adjective).

Reply:
Will change.